kms/createImportJob.js

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main(
+  projectId = 'my-project',
+  locationId = 'us-east1',
+  keyRingId = 'my-key-ring',
+  id = 'my-import-job'
+) {
+  // [START kms_create_import_job]
+  //
+  // TODO(developer): Uncomment these variables before running the sample.
+  //
+  // const projectId = 'my-project';
+  // const locationId = 'us-east1';
+  // const keyRingId = 'my-key-ring';
+  // const id = 'my-import-job';
+
+  // Imports the Cloud KMS library
+  const {KeyManagementServiceClient} = require('@google-cloud/kms');
+
+  // Instantiates a client
+  const client = new KeyManagementServiceClient();
+
+  // Build the parent key ring name
+  const keyRingName = client.keyRingPath(projectId, locationId, keyRingId);
+
+  async function createImportJob() {
+    const [importJob] = await client.createImportJob({
+      parent: keyRingName,
+      importJobId: id,
+      importJob: {
+        protectionLevel: 'HSM',
+        importMethod: 'RSA_OAEP_3072_SHA256',
```

Comment:
A context doc link would be helpful!

Reply:
This shouldn't be inline in the sample, since it's already included in the sample context.
See https://cloud.google.com/kms/docs/importing-a-key#create_importjob